linera-chain/src/data_types.rs

```diff
@@ -1141,11 +1200,9 @@ impl Certificate {
         round: Round,
         mut signatures: Vec<(ValidatorName, Signature)>,
     ) -> Self {
-        if !is_strictly_ordered(&signatures) {
-            // Not enforcing no duplicates, check the documentation for is_strictly_ordered
-            // It's the responsibility of the caller to make sure signatures has no duplicates
-            signatures.sort_by_key(|&(validator_name, _)| validator_name)
-        }
+        // Not enforcing no duplicates, check the documentation for is_strictly_ordered
+        // It's the responsibility of the caller to make sure signatures has no duplicates
+        signatures.sort_by_key(|&(validator_name, _)| validator_name);
```

Comment:
What's happening here? Should we update the comments?

Reply:
Comment is the same as it was before, same for logic. this PR isn't changing it.

Reply:
Right, but we're not using `is_strictly_ordered` anymore, and the comment mentions that.

Reply:
Ups, you're right. IIRC I've removed the usage in all but one place and then forgot to remove comments where necessary.

Reply:
Done in [Review pt2](https://github.com/linera-io/linera-protocol/pull/2745/commits/1c2b92ece23d1d7623718d7ca723563505a14da8).